### PR TITLE
Release: 2022-05-26

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

- Hadoop FDW 2.2.0 release
- Mongo FDW 5.4.0 release
- MySQL 2.8.0 release
- BigAnimal new Auth0 Idp topic

### CODE REVIEW

**Review by jericson-edb — CHANGES_REQUESTED** [review]
I'm not sure about why `&lt;` was used in the Auth0 documentation. It would be good to simplify that, if possible.

File: product_docs/docs/biganimal/release/getting_started/identity_provider/auth0.mdx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+1. In the Create application dialog, enter a name and choose **Native** as the application type. Select **Create**.
+1. On the Connections tab on the page for your application, enable the toggles for the databases containing your user names (Username-Password-Authentication, GitHub, or Google/Gmail).
+1. Select the Settings tab of the page for your application, scroll down, and expand the **Advanced Settings** section.
+1. Select the Certificates tab in Advanced Settings and download the certificate in PEM format.
+1. Select the Addon tab on the page for your application and enable the **SAML2 Web App** toggle. 
+1. Select the Settings tab on the Addon: SAML2 Web App dialog.
+1. Copy the Assertion Consumer Service URL value from BigAnimal and paste it as the Application Callback URL value in Auth0.
+1. Copy the Audience URI from the Set Up Identity Provide page in BigAnimal and paste it as the audience value in the Settings field. The following attributions statements may need modification depending on what form of your user names (for example, if the user names come from GitHub, there are no last name and first name fields):
+
+    - &lt;assertion_path>/name
```
[the remark this answers is not in the document]
Comment (josh-heyer) [reply product_docs/docs/biganimal/release/getting_started/identity_provider/auth0.mdx:28]: In this case, it isn't, strictly-speaking, code (or verbatim text / etc) - the <assertion_path> is a placeholder.

**Review by josh-heyer — APPROVED** [review]

File: product_docs/docs/biganimal/release/getting_started/identity_provider/auth0.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (drothery-edb) [reply product_docs/docs/biganimal/release/getting_started/identity_provider/auth0.mdx:28]: If I format it as code I don't need the %lt; markup. Do you mind if I fix it later? I'd like to get this build published as soon as possible and there are other topics I'll need to change too.

[the remark this answers is not in the document]
Comment (drothery-edb) [reply product_docs/docs/biganimal/release/getting_started/identity_provider/auth0.mdx:28]: Actually I think I'm going to keep it the way it is. We could use code for this topic, but it doesn't work so well for the other sibling topics.

**Pull request merged**